tests/test_type_serialization.py

```diff
@@ -13,3 +16,81 @@ def test_image_serialization():
   b64_str = 'dGVzdCBiYXNlNjQgc3RyaW5n'
   img = Image(value=b64_str)
   assert img.model_dump() == b64decode(b64_str).decode()
+
+
+def test_tool_serialization():
```

Comment:
Is this still useful? It's only testing pydantic serialization now AFAIK

Reply:
Agree - should rm